source/includes/options-mongosqld.yaml

```diff
@@ -690,7 +693,11 @@ args: <service-name>
 description: |
   .. versionadded:: 2.5
 
-  Name of the system service which runs the Kerberized :binary:`~bin.mongosqld`.
+  Registered name of the service using SASL. This option allows you to
```

Comment:
let's just remove the first sentence here (`Registered name of the service using SASL.`)